source/docs/casper/resources/tutorials/beginner/cep78/index.md

```diff
@@ -0,0 +1,326 @@
+For this tutorial, please install an instance of Cep78 using the following
+command and replace the [JSON SCHEMA ESCAPED STRING] with the example
+ESCAPED STRING that's explained in-depth below the command.
+```bash
+casper-client put-deploy \
+  --session-path ./contract/target/wasm32-unknown-unknown/release/contract.wasm \
+  --chain-name casper-test \
+  --node-address http://195.201.167.179:7777 \
+  --secret-key ./secret_key.pem \
+  --payment-amount 300000000000 \
+  --session-arg "collection_name:string='MY_COLLECTION'" \
+  --session-arg "collection_symbol:string='MY_SYMBOL'" \
+  --session-arg "total_token_supply:u64='1000'" \
+  --session-arg "ownership_mode:u8='2'" \
+  --session-arg "nft_kind:u8='2'" \
+  --session-arg "identifier_mode:u8='1'" \
+  --session-arg "nft_holder_mode:u8='2'" \
+  --session-arg "minting_mode:u8='1'" \
+  --session-arg "metadata_mutability:u8='0'" \
+  --session-arg "burn_mode:u8='1'" \
+  --session-arg "nft_metadata_kind:u8='3'" \
+  --session-arg "owner_reverse_lookup_mode:u8='1'" \
+  --session-arg [JSON SCHEMA ESCAPED STRING]
+
+```
+replace [JSON SCHEMA ESCAPED STRING] with the escaped version of
+[above](#custom-metadata-example) example of a custom JSON SCHEMA:
+```
+"json_schema:string='{\"properties\":{\"nft_name\":{\"name\":\"nft_name\",\"description\":\"name_of_nft\",\"required\":true},\"nft_description\":{\"name\":\"nft_description\",\"description\":\"description_of_nft\",\"required\":true},\"nft_url\":{\"name\":\"nft_url\",\"description\":\"url_of_nft\",\"required\":true}}}'"
+```
+Example output:
+```json
+{
+  "id": 414006690162046374,
+  "jsonrpc": "2.0",
+  "result": {
+    "api_version": "1.4.13",
+    "deploy_hash": "a7cdcd4d32b38620090843dd0564418e7219f0d03cb46291b3a4e4fc5a5ea4e1"
+  }
+}
+```
+Use this external [tool](https://jsontostring.com/) to escape a custom
+JSON schema like [above](#custom-metadata-example) or the one from the Cep78 [readme](https://github.com/casper-ecosystem/cep-78-enhanced-nft#example-custom-validated-schema).
+
+_Only escaped JSON metadata can be passed as a session argument when installing the contract or minting an NFT._
+
+## Mint an NFT
+Mint an NFT by calling the previously installed instance of the Cep-78 NFT standard.
+### Query your account
+1. Obtain a `state-root-hash` from the node
+```bash
+casper-client get-state-root-hash --node-address <NODE_ADDRESS>
+```
+Example output:
+```json
+{
+  "id": -7027300282865772761,
+  "jsonrpc": "2.0",
+  "result": {
+    "api_version": "1.4.13",
+    "state_root_hash": "e67fc4806287aff050e12e47afb636e328a24dad3a32170701cd4334fde53fb3"
+  }
+}
+```
+2. Query account state at said `state-root-hash`
+```bash
+casper-client query-state --node-address <NODE_ADDRESS> \
+  --state-root-hash <STATE_ROOT_HASH> \
+  --key <account-hash-YOUR_ACCOUNT_HASH>
+```
+Example:
+```
+casper-client query-state \
+  --node-address http://195.201.167.179:7777 \
+  -s a8b6097502623dbd8a2a8fee17f10e32e46928b2a6f1ed3d01f10aba16140758 \
```

Comment:
Don't abbreviate the state root hash parameter at this point. To do so is confusing and inconsistent. This is basic stuff.